SECURITY.md

```diff
@@ -0,0 +1,5 @@
+# Reporting Security Vulnerabilities
+
+Any potential security vulnerabilities can be reported directly at security@storyblok.com. The Storyblok Team communicates privately and will work on resolving the issues depending on serverity. At the moment there is no bug bounty program.
```

Comment:
@sebgie looks good, there's just a type here `serverity` to fix and we can merge it

Reply:
Eagle-eyed Christian 👍